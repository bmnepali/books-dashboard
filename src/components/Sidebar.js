import React from 'react';

// Components
import Menu from '../components/Menu';
import BookTypes from '../containers/BooksTypes';

const Sidebar = () => {
  return (
    <aside className="col-lg-2 col-sm-3 sidebar">
      <div className="sidebar-sticky">
        {<Menu />}
        {<BookTypes />}
      </div>
    </aside>
  );
};

export default Sidebar;
