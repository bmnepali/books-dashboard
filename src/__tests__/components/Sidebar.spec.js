import * as React from 'react';

// Load Component
import Sidebar from '../../components/Sidebar';

describe('Sidebar Component', () => {
  describe('rendering testing', () => {
    it('renders Sidebar component without any error', () => {
      shallow(<Sidebar />);
    });

    it('matches snapshot', () => {
      const wrapper = shallow(<Sidebar />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
