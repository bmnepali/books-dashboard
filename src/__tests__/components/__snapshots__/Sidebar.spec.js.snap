// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar Component rendering testing matches snapshot 1`] = `
<aside
  className="col-lg-2 col-sm-3 sidebar"
>
  <div
    className="sidebar-sticky"
  >
    <BookList />
    <Connect(BookTypes) />
  </div>
</aside>
`;
